mdx-models/src/main/java/com/mx/path/model/mdx/model/Resources.java

```diff
@@ -308,11 +308,11 @@ private static void registerProfileModelClasses(GsonBuilder builder) {
     // Update Password
     builder.registerTypeAdapter(Password.class, new ModelWrappableSerializer("password"));
     builder.registerTypeAdapter(new TypeToken<Password>() {
-    }.getType(), new ModelWrappableSerializer("update_password"));
+    }.getType(), new ModelWrappableSerializer("password"));
     //UserName
     builder.registerTypeAdapter(UserName.class, new ModelWrappableSerializer("username"));
     builder.registerTypeAdapter(new TypeToken<UserName>() {
-    }.getType(), new ModelWrappableSerializer("update_username"));
+    }.getType(), new ModelWrappableSerializer("username"));
```

Comment:
Same here. We need to add a new model so the (de)serialization continues to work with the old models.